### ISSUE TITLE
Errors with SvelteKit 1.0.0-next.191 or newer

### ISSUE THREAD

**Issue by skrenes** [AlecAivazis/houdini#196]
I hope this isn't an anomaly with my configuration, but since upgrading to [Sveltekit 1.0.0-next.191](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md) or newer, I'm getting a bunch of promise errors that prevent subscriptions from working. Queries still work though, despite the errors. I'm using a similar setup to my [houdini-kit](https://github.com/skrenes/houdini-kit) example.
```
GET /$houdini/index.js net::ERR_ABORTED 403
subscription:1 Uncaught (in promise) TypeError: Failed to fetch dynamically imported module: /src/routes/subscription.svelte
start.js:1190 Uncaught (in promise) TypeError: Failed to fetch dynamically imported module: /src/routes/__layout.svelte
async function (async)
start @ start.js:1183
(anonymous) @ subscription:16
subscription.svelte:29 GET /$houdini/artifacts/SubUsers.js net::ERR_ABORTED 403
```

My guess it may have to do with their [security upgrade to Vite 2.6.12, specify allow list, and print warning](https://github.com/sveltejs/kit/pull/2691) and the location of the `$houdini` folder.

**Comment by AlecAivazis** [comment]
@skrenes did you ever get to the bottom of this? I haven't had a lot of time to dig into the 2.6.12 changes but if you have a working configuration now I can update the project

**Comment by skrenes** [comment]
@AlecAivazis unfortunately I've been swamped and haven't had a chance to try to problem solve this. For my project, I've simply kept `@sveltejs/kit` locked on `1.0.0-next.190`. My guess is that it'll come down to either moving `/$houdini` to `/src/_$houdini` or adding `/$houdini` to the allow list.

**Comment by heliumbrain** [comment]
@skrenes I had a similar issue that I managed to solve by tweaking the Vite conf to this:

```
vite: {
      server: {
        fs: {
          allow: ['..']
        }
      },
			resolve: {
				alias: {
					$houdini: path.resolve('.', '$houdini')
				}
			}
		},
```

Reference: https://vitejs.dev/config/#server-fs-allow

Might be worth trying :)

**Comment by skrenes** [comment]
Thanks so much @heliumbrain ! I also appreciate the reference. I tried to specifically allow only the Houdini folder and config directory, but the allow list seems to only respect paths. So I filed this bug: [https://github.com/vitejs/vite/issues/5689](https://github.com/vitejs/vite/issues/5689). In the meantime, I set the allow line to ```allow: ['.'],``` rather than ```allow: ['..'],```, since it's technically more restrictive.
